### PR TITLE
Split income model in two: Helsinki and other

### PR DESCRIPTION
**Pull request by zptro**
Helsinki model requires new Lauttasaari dummy and storing of mode choice logsum in `ZoneData`.

### CODE REVIEW

**Review by johpiip — APPROVED** [review]

**Review by atte-wsp — CHANGES_REQUESTED** [review]
Test run showed that there is large difference in incomes for Helsinki and other region. For Helsinki result incomes are between 0-5 and for others 0-20000. 

Should there be constant in Helsinki parameters?

File: Scripts/parameters/income.py [hunk 1]
```diff
@@ -1,24 +1,39 @@
...
+        "Hyvinkaa": -0.08642,
+        "Jarvenpaa": -0.13453,
+        "Sipoo": 0.12064,
+        "Tuusula": -0.09369,
+        "Vihti": -0.07904,
     },
+    "standard_deviation": 0.35499,
+}
+log_income_helsinki = {
+    "constant": 0,
```
Comment (atte-wsp) [inline Scripts/parameters/income.py:26]: Is this correct? I think this causes large difference between Helsinki / other.

Comment (zptro) [reply Scripts/parameters/income.py]: Great that you found this error! The constant had indeed disappeared from Lauri's excel file.

**Review by atte-wsp — APPROVED** [review]
Model predictions are ok. Median income is still constantly lower in Helsinki than rest of the region (slides in SharePoint), but I'm not sure if this is error or based on statistics.

File: Scripts/parameters/income.py [hunk 2]
```diff
@@ -1,24 +1,39 @@
 log_income = {
-    "constant": 6.52186,
+    "constant": 6.8611,
     "zone": {
-        "perc_detached_houses_sqrt": 0.16309,
-        "hw_t": 0.07543,
-        "cbd": 0.13188,
+        "perc_detached_houses_sqrt": 0.18717,
```
Comment (atte-wsp) [inline Scripts/parameters/income.py:4]: Check scale as Zonedata defines share as `self["perc_detached_houses_sqrt"] = (100*landdata["detach"]) ** 0.5`

**Review by atte-wsp — CHANGES_REQUESTED** [review]

**Review by atte-wsp — APPROVED** [review]

**Pull request merged**